de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/callbacks/DiskResourceMoveCallback.java

```diff
@@ -34,10 +34,15 @@ public void onSuccess(DiskResourceMove result) {
     @Override
     public void onFailure(Integer statusCode, Throwable caught){
         unmaskCaller();
-        DiskResourceErrorAutoBeanFactory factory = GWT.create(DiskResourceErrorAutoBeanFactory.class);
-        AutoBean<ErrorDiskResourceMove> errorBean = AutoBeanCodex.decode(factory, ErrorDiskResourceMove.class, caught.getMessage());
-
-        ErrorHandler.post(errorBean.as(), caught);
+        try {
+            DiskResourceErrorAutoBeanFactory factory = GWT.create(DiskResourceErrorAutoBeanFactory.class);
+            AutoBean<ErrorDiskResourceMove> errorBean =
+                    AutoBeanCodex.decode(factory, ErrorDiskResourceMove.class, caught.getMessage());
+
+            ErrorHandler.post(errorBean.as(), caught);
+        } catch (Exception e) {
+            ErrorHandler.post(caught);
+        }
```

Comment:
The structure of this try-catch block seems repeated a lot; I don't know if it can be put somewhere to avoid this, but it could be kinder to our future selves to do so.